app/code/community/RicardoMartins/PagSeguro/etc/config.xml

```diff
@@ -164,6 +164,7 @@
                 <active>0</active>
                 <title>Cartão de Crédito - via PagSeguro UOL</title>
                 <order_status>pending</order_status>
+                <paid_status>paid</paid_status>
```

Comment:
A alteração do status do pedido se dá definindo o status padrão para aquele state. A maioria dos módulos de pagamento usa isso como padrão.
Porque um pedido via Pagseguro + Cartão de credito passaria a usar outro status?

Reply:
É muito útil para integrações de pedidos, caso seja necessário utilizar um status personalizado para pedidos pagos.

Ou para visualizações de pedidos no admin personalizadas como o do [Toluca Store.](https://github.com/gamuzatech/tolucastore-magento/blob/master/magento-lts-1.9.4.x/app/code/local/Gamuza/Basic/Block/Adminhtml/Sales/Order/View.php)

Adicionei um fallback para utilizar o status atual caso um novo não for configurado https://github.com/r-martins/PagSeguro-Magento-Transparente/pull/309/commits/8698dd7b49add0ccdaccec146d55234f3d270864